README.md

```diff
@@ -162,6 +162,19 @@ Results in the following JSON Schema files:
 
 </details>
 
+## SourceInfo
+
+The sourceinfo plugin provides a protoc-gen-sourceinfo command that generates an `.sourceinfo.binpb` file, for every `.proto` file.
+Each generated file contains the associated serialized `google.protobuf.SourceInfo` value.
+These files can then be loaded directly or embedded in a binary.
+
+The following helpers are provided for go via `import "github.com/bufbuild/protoschema-plugins/sourceinfo"`:
+
+- `sourceinfo.GlobalFiles()`: A drop-in replacement for protoregistry.GlobalFiles
```

Comment:
How can this be a drop-in if you just are serializing the `google.protobuf.SourceCodeInfo` file?

Reply:
This is hooking into @jhump's library to provide a drop in replacement, backed by normal protoregistry values.